contracts/GaugeController.sol

```diff
@@ -9,22 +9,68 @@ import "./libraries/ExtendedSafeCastLib.sol";
 contract GaugeController is IGaugeController {
     using ExtendedSafeCastLib for uint256;
 
+    /* ================================================================================ */
+    /* Initialization                                                                   */
+    /* ================================================================================ */
+
     struct GaugeInfo {
         uint256 weight;
     }
 
+    /// @notice ERC20 token contract address (used to weight gauges)
     IERC20 public token;
+
+    /// @notice TokenVault for stakers rewards/incentives
     address public rewardVault;
```

Comment:
We should rename it `tokenVault` if we want to go with this name. Or maybe just `vault`. We should ask @asselstine why he decided to name it `TokenVault`.